src/Utilities/TagsModal.js

```diff
@@ -36,10 +37,10 @@ const TagsModal = ({
     const activeSystemTag = useSelector(({ entities, entityDetails }) => entities?.activeSystemTag || entityDetails?.entity);
     const tags = useSelector(({ entities, entityDetails }) => {
         const activeTags = entities?.activeSystemTag?.tags || entityDetails?.entity?.tags;
+
         if (activeTags) {
-            return activeTags?.filter(
-                (tag) => Object.values(tag).some((val) => val?.includes(filterBy))
```

Comment:
Why not just 
```js
val?.toLowerCase().includes(filterBy.toLowerCase())
```

? Does case-insensitive filtering and is far simpler.

Reply:
Hm, you are right on that point, I tried to do that in the first place but couldn't make it to work for me

Reply:
This is pretty much how all (client side) filtering in Compliance works for string searches and it works fine there.